R/resampleClusteredRegions.R

```diff
@@ -77,6 +77,7 @@ resampleClusteredRegions <- function(voxel_df, n_pca = 20,
 
   # extract xyz coords
   xyz <- voxel_df[, 1:3]
```

Comment:
You should get rid of numeric indexing here. This function assumes that the input data is wide, and it should be a long data frame. Please fix and delete the comment.